CHANGELOG.md

```diff
@@ -0,0 +1,44 @@
+## 5.0.1
+   - Relax constraint on march hare to allow 3.x [#44](https://github.com/logstash-plugins/logstash-mixin-rabbitmq_connection/issues/44)
+   - Validate validate :ssl_certificate_password as a password [#43](https://github.com/logstash-plugins/logstash-mixin-rabbitmq_connection/issues/43)
```

Comment:
Everything below is the same content... just formatted as markdown.

Reply:
intentional doubling? `Validate validate`

Reply:
doh! ... i'll sneak the fix directly into master since this has already been merged.